src/parser/view_transform.c

```diff
@@ -1912,7 +1912,7 @@ mq_substitute_subquery_in_statement (PARSER_CONTEXT * parser, PT_NODE * statemen
 				    tmp_result->info.query.q.select.use_merge);
 
 	      assert (query_spec->info.query.orderby_for == NULL);
-	      if (!order_by && query_spec->info.query.order_by)
+	      if (!order_by && query_spec->info.query.order_by && !pt_has_aggregate (parser, tmp_result))
```

Comment:
Even if there is no aggregate function, the order by in the view is not necessary in the main query, right?

Is there a case where order by in the view is needed in the main query?

Reply:
for backwards compatibility.
And in Oracle as well., it was tested to keep the order_by of the view.